lte/gateway/c/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -192,13 +192,18 @@ static int security_mode_t3560_handler(zloop_t* loop, int timer_id, void* arg) {
   OAILOG_INFO(LOG_AMF_APP, "Timer: In security_mode_t3560_handler - T3560\n");
```

Comment:
log statements are not using an unifrom naming convention. it is better to fix it.